components/automate-cli/cmd/chef-automate/generate-a2harb.go

```diff
@@ -0,0 +1,261 @@
+// Copyright © 2017 Chef Software
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"strings"
+
+	ptoml "github.com/pelletier/go-toml"
+)
+
+func readConfigAndWriteToFile() error {
+	//fmt.Printf("reading configs from toml file")
+	initConfigHAPath := initConfigHAPathFlags.path
+	config, err := ptoml.LoadFile(initConfigHAPath)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+	architectureAws := config.Get("architecture.aws")
+	if architectureAws != nil {
+		generateAWSA2HARBFile(config, architectureAws)
```

Comment:
check both the case separately ..
.if aws 
else 
if exsiting-infra
else
invalid config

Reply:
resolved